### PR TITLE
Shaun carmody stretch

### PR DESCRIPTION
**Pull request by shaunmcarmody**

### CODE REVIEW

File: Article/Article.js [hunk 1]
```diff
@@ -1,29 +1,49 @@
...
-- With your selection in place, now chain .forEach() on to the articles variable to iterate over the articles NodeList and create a new instance of Article by passing in each article as a parameter to the constructor.
+- With your selection in place, now chain .forEach() on to the articles variable to 
+  iterate over the articles NodeList and create a new instance of Article by passing in 
+  each article as a parameter to the constructor.
 
 */
 
-let articles;
\ No newline at end of file
+const populateArticle = (data) => {
```
Comment (Mindysueasaurus) [inline Article/Article.js]: This was an awesome addition. You did a great job on this.

File: Form/Form.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+document.querySelector('.form-button').addEventListener('click', (e) => {
+    e.preventDefault();
+    const data = {
+        header: document.querySelector('.form-title').value,
+        date: 'January 30th 2018',
```
Comment (Mindysueasaurus) [inline Form/Form.js]: I was wondering why the date was the 30th.

Comment (shaunmcarmody) [reply Form/Form.js]: I hard coded the date in initially and was supposed to go back in and look for a Date method. IIwill refactor this over the weekend.

Comment (shaunmcarmody) [reply Form/Form.js]:
```suggestion
        date: 'January 30th 2018',
```
`${d.toLocaleDateString('en-GB', { month: 'short' })}
            ${d.getDay()}th,
            ${d.getFullYear()}`

**Pull request merged**